docs/manual/cluster-user/how-to-use-advanced-job-settings.md

```diff
@@ -104,6 +104,91 @@ You can also export and import YAML files using the `Export` and `Import` button
 
 To see a full reference of job protocol, please check [job protocol](https://github.com/microsoft/openpai-protocol/blob/master/schemas/v2/schema.yaml).
 
+## Use Prerequisites
+
+[OpenPAI protocol](https://github.com/microsoft/openpai-protocol/blob/master/schemas/v2/schema.yaml) support users to specify different types of prerequisites (e.g. dockerimage, data, and script) and then reference them in each task role. We consider prerequisites as *sharable* parts in the job protocol. For example, one dataset can be defined as a prerequisite, and can be used by multiple jobs.
+
+Here's one example to use prerequisites:
+
+```yaml
+protocolVersion: 2
+name: test_prerequisites
+type: job
+jobRetryCount: 0
+prerequisites:
+  - type: script
+    name: install-git
+    plugin: com.microsoft.pai.runtimeplugin.cmd
+    callbacks:
+      - event: taskStarts
+        commands:
+          - apt update
+          - apt install -y  git
+  - type: data
+    name: covid-19-data
+    plugin: com.microsoft.pai.runtimeplugin.cmd
+    callbacks:
+      - event: taskStarts
+        commands:
+          - mkdir -p /dataset/covid-19
+          - >-
+            git clone https://github.com/ieee8023/covid-chestxray-dataset.git
```

Comment:
Does this require the `install-git` prerequisite ?

Reply:
No. `require` will be used by marketplace backend automatically. Added a simple comment.